cmd/debos/debos.go

```diff
@@ -253,7 +253,7 @@ func main() {
 		}
 		memsize, err := units.RAMInBytes(options.Memory)
 		if err != nil {
-			fmt.Printf("Couldn't parse memory size: %v\n", err)
+			log.Printf("Couldn't parse memory size: %v\n", err)
```

Comment:
To be honest I'm not sure when I should be using one or the other (log vs fmt). Does golang or go-debos have some guideline when to use which one? 

Looking at fakemachine - it exclusively uses fmt. While debos has a 2:1 ratio for fmt vs log. Should any of the other fmt instanced be also swapped - either here or a separate issue/PR?

Reply:
In debos we should use log; which adds pretty bits e.g. datestamp. For fakemachine, we don't have a log so fmt is fine.

We should transititon to log in debos ;-)